src/gui/optionsdialog.cpp

```diff
@@ -1690,20 +1697,34 @@ void OptionsDialog::initializeStyleCombo()
     const QString prefStyleName = Preferences::instance()->getStyle();
     const QString selectedStyleName = prefStyleName.isEmpty() ? QApplication::style()->name() : prefStyleName;
     QStringList styleNames = QStyleFactory::keys();
-    for (qsizetype i = 1, stylesCount = styleNames.size(); i < stylesCount; ++i)
+
+    if (selectedStyleName.compare(u"system"_s, Qt::CaseInsensitive) == 0)
     {
-        if (selectedStyleName.compare(styleNames.at(i), Qt::CaseInsensitive) == 0)
+        m_ui->comboStyle->addItem(tr("System", "System default Qt style"), u"system"_s);
```

Comment:
`System` -> `System default` or `Qt default`.
How about those suggestions?

Reply:
IMO, if we add "default" to any entry it will confuse users as to what we recommend.

Reply:
>if I don't know in advance what it means.

Honest question: Do you think the tooltip won't help with that?

Reply:
> > if I don't know in advance what it means.
> 
> Honest question: Do you think the tooltip won't help with that?

Maybe. Unless user initially gets a false impression about its meaning and chooses it without looking at tooltip.

Reply:
Anyway, let's not dwell on it. It can be renamed later based on real user feedbacks.